### ISSUE TITLE
Font Library: Consider clarifying which fonts are uploaded vs theme provided in modal

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#55220]
Pulling in feedback from the [FSE Outreach Program's Final Touches call for testing:](https://make.wordpress.org/test/2023/09/18/fse-program-testing-call-26-final-touches/#comment-3184)

>Regarding the Fonts manager having a heading above the external/uploaded fonts. There is a heading that says Theme Fonts and one sees the fonts below it. Nice and clear. After uploading one or more fonts these are added above the Theme fonts section. I missing having a heading above uploaded fonts which clearly tells me what these fonts are. As in:
>
>Heading: Uploaded/External Fonts.
>Fuggles
>etc
>
>Heading: Theme Fonts.
>DM Sans
>IBM Plex Mono
>etc
>
>As these become like sections inside the Fonts Manager Library. Uploaded Fonts and Theme Fonts. Showing which fonts belong to each.

Right now, after uploading a font, it's not clear which comes from the theme vs uploaded separately. There's an argument to be had that this can make it easier for folks to just do what they want to do with fonts without worrying about that kind of separation but I can also see why folks might want to know what came with the theme vs what they are customizing.

**Comment by colorful-tones** [comment]
I added #58403 to reinforce any work that might be done here and surface it in the Global Styles > Typography. This would reduce site builders of having to even launch the Font Library modal to see the different sources of where fonts were installed, e.g. Theme, User, etc.

**Comment by t-hamano** [comment]
#58834 will solve this problem.

**Comment by t-hamano** [comment]
I have added this issue to WP6.5 project board. See [this comment](https://github.com/WordPress/gutenberg/pull/58834#issuecomment-1942963616) for the reason.

**Comment by youknowriad** [comment]
Moved this to 6.6 because it's an enhancement that can land later which will allow us to focus on the main font library issues for 6.5.

**Comment by afercia** [comment]
> having a heading above the external/uploaded fonts.

Totally agree. I think this was also mentioned in other conversatios as well as an accessibility issue as any section of content should be clearly identified by a heading. These are lists of content items. It's not clear why one of the list does have a heading and the other doesn't.

**Comment by t-hamano** [comment]
In the latest Font Library, the text for headings exists in both font lists. Should we change "Installed Fonts" to another text?

![image](https://github.com/WordPress/gutenberg/assets/54422211/76fc1532-f923-4024-a11d-20751d0df7f9)

By the way, accessibility improvements should be addressed by #58834.

**Comment by annezazu** [comment]
I dig this! I think installed fonts works great. I think we can close this out if I'm understanding correctly. Let me know and I can reopen too. At least on my end, this addresses the distinction between the two for me.
